scripts/support/create_ge_images/example/test.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+#
+# Copyright 2015 Telefónica Investigación y Desarrollo, S.A.U
+#
+# This file is part of FI-Core project.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at:
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For those usages not covered by the Apache version 2.0 License please
+# contact with opensource@tid.es
+#
+
+
+# The script is aborted if any command fails. If it is OK that a comand fails,
+# use ./mycomand || true
+set -e
+
+# log into the VM and check the service
+ssh ${USERNAME}@$IP /etc/init.d/apache2 status |grep -q "is running" 
+# connect to webserver 
+wget http://$IP/cgi-bin/mycgi?test -O /dev/null
```

Comment:
jeje, I like it. In fact it could be a solution also to recover the scripts that we have to use to create the image and test it.